shell/egl_window.h

```diff
@@ -112,32 +113,10 @@ class EglWindow : public Egl {
 
   static const struct xdg_surface_listener xdg_surface_listener;
 
-  static void handle_xdg_configure_callback(void* data,
-                                            struct wl_callback* callback,
-                                            uint32_t time);
-
   static const struct wl_callback_listener xdg_configure_callback_listener;
 
   [[maybe_unused]] static struct shm_buffer* next_buffer(EglWindow* window);
 
-  static void paint_pixels_top(void* image,
-                               int padding,
-                               int width,
-                               int height,
-                               uint32_t time);
-
-  static void paint_pixels_bottom(void* image,
-                                  int padding,
-                                  int width,
-                                  int height,
-                                  uint32_t time);
-
-  static void paint_pixels(void* image,
-                           int padding,
-                           int width,
-                           int height,
-                           uint32_t time);
-
```

Comment:
Clean-ups of these do belong in a different patch. These removal have nothing to do with patch. Or if so please why in the commit.